doc/engineering-reference/src/building-system-simulation-system-manager/plant-condenser-loops.tex

```diff
@@ -564,6 +564,64 @@ \subsubsection{Outdoor Drybulb Range Based Operation, Outdoor Wetbulb Range Base
+The supervisory control routine uses the results of building and plant loads at the current timestep to decide if the heat-pump-based hydronic plant should be operated in one of four modes: off, heating-only, cooling-only, or simultaneous-heating-and-cooling. If $\dot Q_{BuildingHeating}$ is less than zero and $\dot Q_{BuildingCooling}$ is also negative (cooling loads are negative) then based on builing loads, the plant is expected to be in cooling-only mode.  if $\dot Q_{BuildingCooling}$ is positive and $\dot Q_{BuildingHeating}$ is postive, then the plant is expected to be in heating-only mode. However, if the outdoor air temperature is lower than the input used in the field called ``Outdoor Low Temperature'' then the heat pump cannot operate in heating and the mode is reset to off. If the building shows both heating and cooling loads, when $\dot Q_{BuildingHeating}$ is postive and $\dot Q_{BuildingCooling}$ is negative, then the plant is expected to be in simultaneous-heating-and-cooling mode.  In the case of a very small plant with only a single heat pump that cannot operate in simultaneous mode, the plant is set to cooling-only or heating-only depending on which has the more dominant load. However, if the outdoor air temperature is lower than the input used in the field called ``Outdoor Low Temperature'' then the heat pump cannot operate in heating and the mode is reset to cooling-only. After the building loads have been used, additional changes in mode are considered using the current plant loads. If the building loads indicate that the plant is expected to be in heating-only mode but the current primary cooling hydronic plant load is negative, $\dot Q_{LoopDemand, ChW} < 0.0$, then the mode is changed to simultaneous-heating-and-cooling (if available). Similarly, if the building loads indicate that the plant is expected to be in cooling-only mode but the current primary heating hydronic plant load is positive, $\dot Q_{LoopDemand, HW} > 0.0$, and the outdoor air temperature is higher than the minimum limit, then the mode is changed to simultaneous-heating-and-cooling (if available).  The selected mode is reported to the output variable called ``Supervisory Plant Heat Pump Operation Mode.''
```

Comment:
One minor issue for the documentation here: in the TeX text file, the paragraphs need an actual blank line (or one extra line break, like Line 578 above) between paragraphs. Otherwise these 3-4 paragraphs here (from Line 609 to Line 613, for example) would all cramped into one long paragraph in the pdf file compiled.